api/v1alpha1/baseboardmanagement_types.go

```diff
@@ -64,6 +64,10 @@ type Connection struct {
 	// +kubebuilder:validation:MinLength=1
 	Host string `json:"host"`
 
+	// Port is the IPMI port of the BaseboardManagement.
+	// +kubebuilder:validation:MinLength=1
+	Port string `json:"port"`
```

Comment:
Can this be an Int?

Reply:
I kept it as string because bmclib expects port as a `string`. But we can change it to `int` if that makes more sense 👍